test/test.tools.js

```diff
@@ -51,36 +61,53 @@ describe("Monkeypatching", () => {
 describe("Test tools for parsing new gmail body_params", () => {
     const gmail = new Gmail();
     const data =  JSON.parse(testData.new_gmail_archive_action_body_params);
+    const threads = gmail.tools.extract_from_graph(data, gmail.check.data.is_thread);
+    const threadData = threads.map(thread => gmail.tools.get_thread_data(thread))[0];
```

Comment:
This value is only used in a single test, and should be put inside that test for improved clarity.

Reply:
This could probably be simplified to `const threadData = gmail.tools.get_thread_data(threads[0]);` or am I missing something?

Reply:
maybe you are right I wanted to repeat logic from code)